designer/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/component/package.scala

```diff
@@ -45,6 +55,30 @@ package object component {
     def sortMethod(component: ComponentListElement): (String, String) = (component.name, component.id.value)
   }
 
+  implicit val allowedProcessingModesEncoder: Encoder[AllowedProcessingModes] = Encoder.instance {
+    case AllowedProcessingModes.AllProcessingModes            => ProcessingMode.all.asJson
```

Comment:
We have Enumeratum in scope. WDYT about improving this `all` method? At the moment, we have to remember to add the new case to the `all` val.